### PR TITLE
Commits for #564, #566 and #554

### LINKED ISSUE #564: WebUI - add prerequisites to README
A proposal bringing some colors. ;-)

image

### PR DESCRIPTION
**Pull request by claudioantonio**
fixes #564 #554 #566

### CODE REVIEW

File: webui/Readme.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # git-bug rich web UI
 
+## Prerequisites
+
+<div align="center">
+<img height="119px" alt="ReactJS" src="https://www.pinclipart.com/picdir/big/537-5374089_react-js-logo-clipart.png">
+<img width="150px" src="https://material-ui.com/static/logo_raw.svg">
```
Comment (MichaelMure) [inline webui/Readme.md:7]: Can you download and commit those images (with a simple name) so that the repo stay self-contained?

**Review by MichaelMure — CHANGES_REQUESTED** [review]

**Comment by MichaelMure** [comment]
Also, the CI fail on a typescript code:
```
 /home/runner/work/git-bug/git-bug/webui/src/components/Author.tsx
TypeScript error in /home/runner/work/git-bug/git-bug/webui/src/components/Author.tsx(14,15):
Property 'email' does not exist on type '({ __typename?: "Identity" | undefined; } & Pick<Identity, "[object Object]">) | ({ __typename?: "Identity" | undefined; } & Pick<Identity, "[object Object]">) | ... 10 more ... | ({ ...; } & Pick<...>)'.
  Property 'email' does not exist on type '{ __typename?: "Identity" | undefined; } & Pick<Identity, "[object Object]">'.  TS2339

    12 | 
    13 | const Author = ({ author, ...props }: Props) => {
  > 14 |   if (!author.email) {
       |               ^
    15 |     return <span {...props}>{author.displayName}</span>;
    16 |   }
    17 |
```

File: webui/Readme.md (hunk 1 above)
Comment (claudioantonio) [reply webui/Readme.md:7]: For sure!

**Comment by claudioantonio** [comment]
I will split this PR: PR1 for #564 and #566 and PR2 for #554.
Divide and conquer! ;-)

**Pull request closed without merging**